rust/tools/authorization-logic/third_party/antlr4-rust/LICENSE.txt

```diff
@@ -0,0 +1,26 @@
+[The "BSD 3-clause license"]
```

Comment:
@conradgrobler I moved the antl4rust jar to a third_party/antlr4-rust/ directory and copied the LICENSE.txt from that project.